### PR TITLE
Feat ✨:  Added an endpoint to download video locally 

### PR DESCRIPTION
**Pull request by DestinedCodes**
​
## Description

I added an extra endpoint to allow users downlaod videos to their local device. It is exactly same as streaming a video but here i specified filetype and this makes FastAPI send us the video as a diwnlaodsble rather than just rendering it as we did before
​
## Related Issue (Link to linear ticket)

[ Linear Ticket ](https://linear.app/zuri-project-backend/issue/HEL-1/create-video-disbursing-endpoint>, this is kinda unrelated to my ticket which i have already fulfilled.
​
## Motivation and Context

​I am adding this endpont because it is needed in cases where the user would want to download the video locally than just play it in the browser.

## How Has This Been Tested?

I have tested this locally on my PC and mobile decide so when i try to access the url, download begins automatically 
​
## Screenshots (if appropriate - Postman, etc):
https://github.com/hngx-org/helpmeout-api/assets/84413505/5613709a-58b8-476b-9226-24eff73f776c

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
​
## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by zxenonx — CHANGES_REQUESTED** [review]
Thanks @DestinedCodes

File: app/routes/video_routes.py [hunk 1]
```diff
@@ -284,6 +284,37 @@ def stream_video(video_id: str, db: Session = Depends(get_db)):
 
     return FileResponse(video.original_location, media_type="video/mp4")
 
+@router.get("/download/{video_id}.mp4")
```
[the remark this answers is not in the document]
Comment (DestinedCodes) [reply app/routes/video_routes.py:287]: Actually the .mp4 in the url doesn't affect the downlaod format, but that'll be fixed it doesn't need to be a part if the url.

[the remark this answers is not in the document]
Comment (zxenonx) [reply app/routes/video_routes.py]: Let's just remove it then please @DestinedCodes

**Review by zxenonx — APPROVED** [review]
Thanks @DestinedCodes 
Good job

**Comment by DestinedCodes** [comment]
Thanks for the wonderful review. All requested changes made

**Pull request merged**